apps/settings/test/unit/panels/screen_lock_passcode/screen_lock_passcode_test.js

```diff
@@ -39,6 +56,7 @@ suite('ScreenLockPasscode > ', function() {
           realMozSettings = window.navigator.mozSettings;
           window.navigator.mozSettings = MockNavigatorSettings;
 
+          PasscodeHelper = PasscodeHelperModule;
```

Comment:
I'm not sure I fully understand, but if I remove this line, I get `TypeError: object is undefined` errors for those source lines that mock out the check function (i.e. line 126 in my latest commit).